internal/providers/terraform/ibm/testdata/pi_instance_test/pi_instance_test.golden

```diff
@@ -0,0 +1,36 @@
+
+ Name                                            Monthly Qty  Unit      Monthly Cost 
+                                                                                     
+ ibm_pi_instance.aix-shared-s922-instance                                            
```

Comment:
As discussed can you:
- change the usage up to take a montly instance hours param?
- make test cases that don't have the instance hours params supplied?

Reply:
noted